scaaml/capture/aes/acqktp.py

```diff
@@ -14,9 +14,11 @@
 """Generating uniformly distributed keys."""
 
 import random
-from typing import Iterable, Optional
+from typing import Generator, Iterable, Optional, Tuple
```

Comment:
NIT (if you decide to accept the NIT below :))
```suggestion
from typing import Iterable, Iterator, Optional, Tuple
```

Reply:
Done